src/EmuFs.cc

```diff
@@ -72,6 +72,22 @@ void EmuFile::update(dev_t device, ino_t inode, uint64_t size) {
   size_ = size;
 }
 
+#ifndef MFD_CLOEXEC
+#define MFD_CLOEXEC 0x0001u
+#endif
+#ifndef MFD_ALLOW_SEALING
+#define MFD_ALLOW_SEALING 0x0002u
+#endif
+
+#ifndef __NR_memfd_create
+#define __NR_memfd_create 319
+#endif
+
+static int memfd_create(const char *__name, unsigned int __flags)
+{
+  return syscall(__NR_memfd_create, __name, __flags);
+}
```

Comment:
Call this `rr_memfd_create` so it doesn't conflict with a libc version when that appears.